RUFAS/current_day_conditions.py

```diff
@@ -57,23 +58,65 @@ def __post_init__(self):
             self.rainfall = self.precipitation
 
     @staticmethod
-    def determine_daylength(month: int) -> int:
+    def determine_daylength(day_number: int, geographic_latitude_radians: float, month: int,
+                            angular_velocity=0.2618) -> float:
         """
-        Approximate day length of the month by using data from Madison, WI.
+        Calculates the day length for the field based on its day and latitude
 
         Parameters
         ----------
-        month: int
-            Month of the year.
+        day_number : int
+            Day number of the year
+        geographic_latitude_radians : float
+            geographic latitude in radians
+        angular_velocity: float default = 0.2618
+            angular velocity of earth
+        month : int
+            month of the simulation
 
         Returns
         -------
-        int
-            Day length of the month (hour).
+        float
+            The day length of  the field on specific day
```

Comment:
```suggestion
            The length of the current day (hours).
```
This day length is not specific to any field.